pkg/asset/operators/operators.go

```diff
@@ -0,0 +1,68 @@
+package operators
+
+import (
+	"github.com/openshift/installer/pkg/asset"
+	"path/filepath"
+)
+
+// operators generates the dependent operator config.yaml files
+type operators struct {
+	assetStock Stock
+	directory  string
+}
+
+var _ asset.Asset = (*operators)(nil)
+
+type genericData map[string]string
+
+// Dependencies returns all of the dependencies directly needed by an
+// operators asset.
+func (o *operators) Dependencies() []asset.Asset {
+	return []asset.Asset{
+		o.assetStock.ClusterVersionOperator(),
+		o.assetStock.KubeCoreOperator(),
+		o.assetStock.NetworkOperator(),
+	}
+}
+
+// Generate generates the respective operator config.yml files
+func (o *operators) Generate(dependencies map[asset.Asset]*asset.State) (*asset.State, error) {
+	//cvo := dependencies[o.assetStock.ClusterVersionOperator()].Contents[0]
+	kco := dependencies[o.assetStock.KubeCoreOperator()].Contents[0]
+	no := dependencies[o.assetStock.NetworkOperator()].Contents[0]
+	tnco := dependencies[o.assetStock.Tnco()].Contents[0]
+	//ingress := dependencies[o.assetStock.IngressOperator()].Contents[0]
+	addon := dependencies[o.assetStock.KubeAddonOperator()].Contents[0]
+
+	// kco+no+tnco go to kube-system config map
+	kubeSys, err := configMap("kube-system", "cluster-config-v1", genericData{
```

Comment:
This asset generate seems to be just `cluster config`, Can we have these configs be separate assets?